lib/uwb/UwbSession.cxx

```diff
@@ -61,8 +59,8 @@ UwbSession::StopRanging()
     PLOG_VERBOSE << "stop ranging";
     bool rangingActiveExpected = true;
     const bool wasRangingActive = m_rangingActive.compare_exchange_weak(rangingActiveExpected, false);
-    if (!wasRangingActive) {
-        return;
+    if (wasRangingActive) {
+        //StopRangingImpl();
```

Comment:
Thanks for fixing this. Any reason this function call is commented out?

Reply:
The function call wasn't there to begin with, but that's where it would go, so I just added it but commented it out. I'll get back to it when I add the code to invoke the StopRanging IOCTL